components/lcd_touch/esp_lcd_touch/esp_lcd_touch.c

```diff
@@ -195,3 +199,27 @@ esp_err_t esp_lcd_touch_del(esp_lcd_touch_handle_t tp)
 
     return ESP_OK;
 }
+
+esp_err_t esp_lcd_touch_register_interrupt_callback(esp_lcd_touch_handle_t tp, esp_lcd_touch_interrupt_callback_t callback)
+{
+    esp_err_t ret = ESP_OK;
+    assert(tp != NULL);
+    assert(callback != NULL);
```

Comment:
Could we disable the interrupt in case user passes callback==NULL?

Reply:
I changed it. Now, it is possible :-)